frontend/packages/ceph-storage-plugin/src/__tests__/independent-mode/breakdown-card.spec.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
```

Comment:
Why do we have a structure like:

```
$ tree packages/ceph-storage-plugin/src/components/independent-dashboard-page
packages/ceph-storage-plugin/src/components/independent-dashboard-page
|-- breakdown-card
|   `-- card.tsx
|-- details-card
|   `-- card.tsx
|-- status-card
|   `-- card.tsx
`-- utilization-card
    `-- card.tsx
```

I'd suggest to simplify it to:

```
|-- breakdown-card.tsx
|-- details-card.tsx
|-- status-card.tsx
`-- utilization-card.tsx
```

Also, please keep the path convention for better test vs. tested-unit correlation, by placing this spec at

```
src/__tests__/independent-dashboard-page/breakdown-card.spec.tsx
```

Reply:
I see that you're following a single-test-directory convention in Ceph plugin:

```
$ find packages/ceph-storage-plugin/src -type d -name __tests__
packages/ceph-storage-plugin/src/__tests__
```

in which case it's more important to keep your `__tests__` structure well organized, in my opinion.